.ci/jenkins/Jenkinsfile.promote

```diff
@@ -139,7 +139,7 @@ pipeline {
             }
         }
 
-        stage('Upload OptaPlanner documentation and distribution') {
+        stage('Upload OptaPlanner documentation') {
```

Comment:
Javadocs and the manual are no longer a part of the distribution. But we need to upload them and they still reside in the optaplanner repository.

Reply:
Isn't that part produces optaplanner-distribution-internal?

Reply:
I don't think so; this stage only uploads the distribution and documentation to the JBoss filemgmt web server. The internal distribution is created as a part of the maven build and should be uploaded to Nexus at this step: https://github.com/kiegroup/optaplanner/blob/ab3a6a5b8895c0def7d41faf83fea39ff16878ba/.ci/jenkins/Jenkinsfile.deploy#L181.

Reply:
Thanks